website/content/en/docs/building-operators/golang/operator-scope.md

```diff
@@ -89,42 +97,55 @@ in `role_binding.yaml` are used to grant the operator permissions to access and
 and `auth_proxy_*.yaml` are not relevant to changing the operator's resource permissions.
 
-### Changing the permissions
+### Changing the permissions to Namespaced
 
 To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use `Role`s
-and `RoleBinding`s instead of `ClusterRole`s and `ClusterRoleBinding`s, respectively.
-
-[`RBAC markers`][rbac-markers] defined in the controller (e.g `controllers/memcached_controller.go`)
-are used to generate the operator's [RBAC ClusterRole][rbac-clusterrole] (e.g `config/rbac/role.yaml`). The default
- markers don't specify a `namespace` property and will result in a `ClusterRole`.
+=======
 
-Update the RBAC markers to specify a `namespace` property so that `config/rbac/role.yaml` is generated as a `Role`
- instead of a `ClusterRole`.
+- Inform the Namespace to the [Manager][ctrl-manager] 
 
-Replace:
+By default, the [Manager][ctrl-manager] does not have any namespace specified in `main.go`, and hence it will watch all the namespaces. In order to restrict the controllers to watch a specific namespace, specify it while creating the manager. Update the `NewManager` to inform the Namespace, in our `Memcahced` example it would like: 
 
 ```go
-// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/status,verbs=get;update;patch
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+    Namespace:          "memcached-operator-system", // operator namespace
+})
+...
 ```
 
-With namespaced markers:
+- Change the scope of the RBAC permissions from cluster-wide to a specific namespace
+
+You will need to use `Role`s and `RoleBinding`s instead of `ClusterRole`s and `ClusterRoleBinding`s, respectively.
+
+[`RBAC markers`][rbac-markers] defined in the controller (e.g `controllers/memcached_controller.go`)
+are used to generate the operator's [RBAC ClusterRole][rbac-clusterrole] (e.g `config/rbac/role.yaml`). The default
+ markers don't specify a `namespace` property and will result in a `ClusterRole`.
+
+Update the [`RBAC markers`][rbac-markers] in `<kind>_controller.go` with `namespace=<namespace>` where the `Role` is to be applied, such as:
 
 ```go
-// +kubebuilder:rbac:groups=cache.example.com,namespace="my-namespace",resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=cache.example.com,namespace="my-namespace",resources=memcacheds/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=cache.example.com,namespace=memcached-operator-system,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
```

Comment:
The `namespace` needs to be the ns which will be applied the roles. We are using the tutorial exampled here and in this case it is `memcached-operator-system`. Note that when we run `make deploy IMG=` for the tutorial it will create the `memcached-operator-system` and deploy the operator on it. So, the roles need to be applied to `memcached-operator-system`